src/load.js

```diff
@@ -27,6 +14,19 @@ define(['ModernizrProto'], function(ModernizrProto) {
     };
   }
 
+  /**
+   * Previously, Modernizr.load was an alias for yepnope. Since yepnope was
+   * deprecated, we removed it as well. It is not available on the website builder,
+   * this is only included as an improved warning to those who build a custom
+   * version locally.
+   *
+   * @memberof Modernizr
+   * @name Modernizr.load
+   * @access private
```

Comment:
this is not a private api

Reply:
you thought otherwise in https://github.com/Modernizr/Modernizr/commit/9b9903953a0415ddcb09d663a85efb1830eb55ba#diff-749fde5b44f91ad90a2478a03e8e8bd6 :-) probably a copyPaste oversight?

Reply:
will remove the private tag of course